kitsune/sumo/utils.py

```diff
@@ -328,3 +329,16 @@ def is_ratelimited(request, name, rate, method=['POST'], skip_if=lambda r: False
             Record.objects.info('sumo.ratelimit', '{key} hit the rate limit for {name}',
                                 key=key, name=name)
     return request.limited
+
+
+def get_browser(request):
```

Comment:
Utils functions should have docstrings that explain what they do, and the arguments they take.

Also, I think this function would be easier to test if took a user agent string instead of a request object.